event_webhook_types.go

```diff
@@ -962,3 +962,21 @@ type WikiPageEvent struct {
 		Action  string `json:"action"`
 	} `json:"object_attributes"`
 }
+
+// SubGroupEvent represents a subgroup event.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/user/project/integrations/webhook_events.html#subgroup-events
+type SubGroupEvent struct {
+	CreatedAt      string `json:"created_at"`
+	UpdatedAt      string `json:"updated_at"`
```

Comment:
Both of these should be `*time.Time`

Reply:
Thanks, wasn't sure what to use since both variants are currently used - changed